### PR TITLE
Iss 104

### PR DESCRIPTION
**Pull request by aliskycloud**
I don't know why it doesn't return the browser name in the html page

### CODE REVIEW

File: firstapp/home/views.py [hunk 1]
```diff
@@ -7,3 +7,12 @@
 def say_hello(request):
     return HttpResponse('Hello World Django')
 
+
+def browser_info(request):
+    user_agent = ''
+    if request.method == 'GET':
+        user_agent = request.GET.get('user_agent', '')
```
Comment (millad90s) [inline firstapp/home/views.py:14]: !!! what is request.Get.get() !!!!

instead try request.META['HTTP_USER_AGENT'])

Comment (aliskycloud) [reply firstapp/home/views.py]: I used this code and it worked. thank you

Comment (aliskycloud) [reply firstapp/home/views.py]: I created the '.gitignore' and pushed it again

**Review by millad90s — APPROVED** [review]
for now it can be approved, we can improve later

**Pull request merged**